webapp/app/forms/steps/steuerlotse_step.py

```diff
@@ -60,6 +60,7 @@ def prepare_render_info(cls, stored_data, *args, **kwargs):
             step_intro=ngettext(cls.intro, cls.intro_multiple, num=cls.number_of_users(stored_data))
             if cls.intro_multiple else cls.intro,
             form=None,
+            stored_data=stored_data,
```

Comment:
I was thinking about the following question:

Couldn't we adapt the `prepare_render_info` method of the FormSteuerlotseStep when the stored_data is already set here (directly updating render_info.stored_data instead of stored_data)?

I came to the conclusion that it is fine that way because we want to update the stored data to (maybe we should set a comment for this). Do you see that differently?

Reply:
I think that's actually a great idea. Of course, if works like it is now, but if we change something more meaningfully in the super method (SteuerlotseStep), it will not be propagated to the sub-class (FormSteuerlotseStep).

Reply:
Did you mean it like I did it in 31cc427?

Reply:
Yes, that's what I meant. However, we need to check that this does not break anything else because the side effect of updating the stored data is now gone.

Reply:
`stored_data` is a local variable to the `get_correct_step` method of the StepChooser and is not accessed after calling `prepare_render_info`. Is that what you mean or did I get you wrong?

Reply:
Exactly. And that is the only point where we call `prepare_render_info`, correct? Then that is exactly what I meant and that is fine :)